Marlin/Marlin_main.cpp

```diff
@@ -3298,6 +3410,45 @@ inline void gcode_G28() {
         } //xProbe
       } //yProbe
 
+      #if ENABLED(PROBE_FAIL_PANIC)
+        if(reprobe_attempts == NUM_ATTEMPTS && probePointCounter == -1)
+        {
```

Comment:
The opening parenthesis for a block should be on the same line. Add a blank line if it helps improve the readability of complex code. See https://github.com/MarlinFirmware/Marlin/wiki/DNE-Coding-Standards